### DISCUSSION TITLE
Have any way to use `alt + ←/→` to move between panes when lock in enabled(i.e. `Ctrl+g`).

### DISCUSSION THREAD

**Discussion by Squirre17** [zellij-org/zellij#2615] [Q&A] [answered]
I want to use `Alt + Left/Right Arrow` to easily switch between panes, but I also need to lock Zellij because there are conflicts with some keys.

**Comment by eemilhaa** [comment, marked as answer]
You can bind the keys specifically for the `locked` mode in your `config.kdl`.

For example:

```
keybinds {
    locked {
        bind "Alt Left" { MoveFocus "Left"; }
        bind "Alt Right" { MoveFocus "Right"; }
        bind "Alt Down" { MoveFocus "Down"; }
        bind "Alt Up" { MoveFocus "Up"; }
    }
}
```

The [documentation](https://zellij.dev/documentation/keybindings.html) has some nice examples too.
